cmd/ocsp-updater/main.go

```diff
@@ -268,6 +273,73 @@ func (updater *OCSPUpdater) oldOCSPResponsesTick(batchSize int) {
 	updater.generateOCSPResponses(statuses)
 }
 
+func (updater *OCSPUpdater) getSerialsIssuedSince(since time.Time, batchSize int) ([]string, error) {
```

Comment:
We seem to sleep and wait even if there are more than LIMIT available to be worked on. 

We should check for that occuring and call this again immediately.

I imagine this change also might have to happen elsewhere.